v4/go.sum

```diff
@@ -1,3 +1,5 @@
+github.com/IBM/go-sdk-core v1.1.0 h1:pV73lZqr9r1xKb3h08c1uNG3AphwoV5KzUzhS+pfEqY=
```

Comment:
Not sure why your environment somehow references a really OLD version of the go core, but this change should not be merged in.